spikeinterface/sorters/runsorter.py

```diff
@@ -42,34 +66,49 @@
     """
 
 
-def run_sorter(sorter_name, recording, output_folder=None,
-               remove_existing_folder=True, delete_output_folder=False,
-               verbose=False, raise_error=True,
-               docker_image=None, singularity_image=None,
-               with_output=True, **sorter_params):
+def run_sorter(
+    sorter_name: str,
+    recording: BaseRecording,
+    output_folder: str = None,
+    remove_existing_folder: bool = True,
+    delete_output_folder: bool = False,
+    verbose: bool = False,
+    raise_error: bool = True,
+    docker_image: bool = False,
+    singularity_image: bool = False,
```

Comment:
Shouldn't these be `Union[str, bool, None]`?

Reply:
@alejoe91 fixed